src/test/EigenPod.t.sol

```diff
@@ -1885,4 +1885,192 @@ contract EigenPodTests is ProofParsing, EigenPodPausingConstants {
     ) public view {
         BeaconChainProofs.verifyWithdrawal(beaconStateRoot, withdrawalFields, proofs);
     }
- }
\ No newline at end of file
+ }
+
+
+//TODO: Integration Tests from old EPM unit tests:
+// TODO: salvage / re-implement a check for reentrancy guard on functions, as possible
```

Comment:
General question, when we're commenting out these tests in several places, whats the general goal for the re-implementation?

Reply:
Rewrite the tests for clarity per the EigenLayer testing guide

The tests above will be reimplemented once we begin writing integration tests

Reply:
Okay, after discussion in standup going to move this to a combined EigenPodManager-Pod unit test file...